test/suite/helper_tiered.py

```diff
@@ -51,20 +51,19 @@ def get_auth_token(storage_source):
         auth_token = os.getenv('GOOGLE_APPLICATION_CREDENTIALS')
     return auth_token
 
-# Get buckets configured for the storage source
-
-# S3 buckets with their regions
-s3_buckets = ['s3testext-us;us-east-2', 's3testext;ap-southeast-2']
-
-# Local buckets do not have a region
-local_buckets = ['bucket1', 'bucket2']
+# Buckets configured for the storage source.
+buckets = {
+    # S3 buckets requires a region.
+    "s3_store": ['s3testext-us;us-east-2', 's3testext;ap-southeast-2'],
+    "dir_store": ['bucket1', 'bucket2'],
+    "gcp_store": ["gcptestext-us-jie", "gcptestext-ap-jie"],
+    "azure_store": ["azuretestext-us", "azuretestext-ap"]
+}
 
 # Get name of the bucket at specified index in the list.
 def get_bucket_name(storage_source, i):
-    if storage_source == 's3_store':
-        return s3_buckets[i]
-    if storage_source == 'dir_store':
-        return local_buckets[i]
+    if storage_source in buckets:
+        return buckets[storage_source][i]
```

Comment:
Would it be simpler to say `return buckets[storage_source][i]`? And just let it throw an exception/fail the test if it's not in the list.

Reply:
Done! [0b9a3d0](https://github.com/wiredtiger/wiredtiger/pull/8927/commits/0b9a3d065ebbea92e885c10c3934e94857660dcb)